mkdocs.yml

```diff
@@ -6,31 +6,31 @@ nav:
 - Introduction: index.md
 - Installation: installation.md
 - Usage: usage.md
-- Building and deploying for cf.gov:
-    - New projects: new-projects.md
-    - Branching and merging: branching-merging.md
-    - Feature flags: feature-flags.md
-    - Split testing: split-testing.md
-    - How we use Travis CI: travis.md
-- Working with Wagtail:
+- Pages and Components:
+    - Atomic structure and design: atomic-structure.md
     - Creating and editing components: editing-components.md
-- Back-end guidelines:
-    - Django and Wagtail migrations: migrations.md
-    - Wagtail pages: wagtail-pages.md
     - Forms in Wagtail page context: forms-in-wagtail.md
-    - Testing: testing-be.md
-    - Translation: translation.md
+    - Wagtail pages: wagtail-pages.md
+    - Django and Wagtail migrations: migrations.md
```

Comment:
Is this last item purposefully non-alphabetical?

Reply:
Nope.

Reply:
Fixed in d3c3d7b.